admin_guide/node-allocatable-resources.adoc

```diff
@@ -0,0 +1,58 @@
+= Node Allocatable resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+To provide more reliable scheduling and minimize node resource overcommitment,
+each node has a chance to specify how much resources get consumed
+by kubernetes related components (e.g. kubelet, kube-proxy, docker) running on a node
```

Comment:
Should we use the naming from the systemd units? If so, it's atomic-openshift-node for OSE/AP and origin-node for Origin